### PR TITLE
Update program areas page to reflect correct mini project cards

### LINKED ISSUE #1614: Replace incorrect mini project cards in Program Areas page with correct ones
### Overview
We need to replace the incorrect mini project cards in the Program Areas page with the correct ones, so that the page is ready to share.

### Action Items
- [x] Replace all of the mini project cards in the Program Areas page that are not consistent with the ones in the Program Areas [Figma file](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=4452%3A21046), adding in the correct ones in their place

### Resources/Instructions
[Program Areas page](https://www.hackforla.org/program-areas)
[Program Areas Figma](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=4452%3A21046)

Figma file mini project cards (correct):
Screen Shot 2021-05-30 at 2 43 35 AM

Current Program Areas page mini project cards (incorrect)
Screen Shot 2021-05-30 at 2 43 42 AM

### PR DESCRIPTION
**Pull request by arghmatey**
Fixes #1614 

Replaced incorrect project card information to match the Program Areas page Figma file.

### CODE REVIEW

**Review by jbubar — CHANGES_REQUESTED** [review]
Great job so far on this pr.
I noticed the the project-in-formation.png image has not been added to the repo... Is that right?
could you add it?

**Comment by arghmatey** [comment]
@jbubar Added placeholder image to the repo and updated any cards that had the old placeholder image on them. 

Thank you for helping me!

File: _data/internal/program-areas/justice-card.yml [hunk 1]
```diff
@@ -28,8 +28,8 @@ projects:
     image_alt: Programming code icon that reads helping the homeless with code.
     link: https://www.hackforla.org/projects/heart
   - id: 00000000
-    name: Racial Justice
-    image: /assets/images/projects/placeholder.jpg
+    name: '#racial-justice'
+    image: /assets/images/projects/project-in-formation.png
```
Comment (jbubar) [inline _data/internal/program-areas/justice-card.yml:32]: this is not the right path. it looks like the image is in /assets/images/projects/project-in-formation.png

**Review by jbubar — CHANGES_REQUESTED** [review]

File: _data/internal/program-areas/justice-card.yml (hunk 1 above)
Comment (arghmatey) [reply _data/internal/program-areas/justice-card.yml:32]: @jbubar I believe that's the path I have currently in the commit. Can you double-check and point me in the right direction if not?

Comment (jbubar) [reply _data/internal/program-areas/justice-card.yml:32]: OMG! hahhahaha. what did I write? So sorry about that. not sure what I was thinking.

Comment (jbubar) [reply _data/internal/program-areas/justice-card.yml:32]: This is perfect

**Pull request merged**